### PR TITLE
VP Delegated To User card - WIP

### PR DESCRIPTION
**Pull request by 1emu**
### Looking at an address different than the logged user

- NO VP deleted to them && NO VP delegated to another user
image

- NO VP delegated to them && VP delegated to another user
image

- VP delegated to them && NO VP delegated to another user
image
 
- VP delegated to them && VP delegated to another user
image

### Looking at the logged user address (mocked)

- NO VP deleted to user && NO VP delegated to another user
image

- NO VP delegated to user && VP delegated to another user
image

- VP delegated to user && NO VP delegated to another user
 image

- VP delegated to user && VP delegated to another user (edited as if we should only display 3 candidates instead of 5)
image

### See All button opens Delegated VP list sorted by VP
image

### Mobile View

image

image

image

image

image

image

### CODE REVIEW

**Comment by 1emu** [comment]
Update:

Remove "Delegate VP" button when looking at another user's profile that hasn't delegated his VP
image

Still shows on user profile
image

Remove "Read more" until we figure out the link

File: src/components/Delegation/DelegatedToUserCard.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
+import { Card } from 'decentraland-ui/dist/components/Card/Card'
+import { Stats } from 'decentraland-ui/dist/components/Stats/Stats'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { DelegationResult } from '../../hooks/useDelegation'
+import './DelegatedToUserCard.css'
+import VotingPowerListItem from './VotingPowerListItem'
+import { VotingPowerList } from './VotingPowerList'
```
Comment (andyesp) [inline src/components/Delegation/DelegatedToUserCard.tsx:13]: I'd rename this to VotingPowerListModal so we can detect easily this is a modal 🤔 What do you think?

File: src/hooks/useProfile.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import useAsyncMemo from "decentraland-gatsby/dist/hooks/useAsyncMemo";
+import profiles, { Profile } from 'decentraland-gatsby/dist/utils/loader/profile'
+
+
+export default function useProfile(address?: string | null) {
+  const [profile, profileState] = useAsyncMemo<Profile>(() => profiles.load(address || ''), [address])
+  const isProfile = !!profile && !profile.isDefaultProfile
+  return {profile, profileState, isProfile}
```
[the remark this answers is not in the document]
Comment (1emu) [reply src/hooks/useProfile.ts:8]: Decided with @andyesp  rename this to isDclAvatar

File: src/components/Delegation/DelegatedFromUserCard.tsx [hunk 3]
```diff
@@ -10,53 +10,54 @@ import Scale from '../Icon/Scale'
...
 import DelegatedCardProfile from './DelegatedCardProfile'
-import Empty from '../Proposal/Empty'
-import './DelegatedCard.css'
+import Empty from '../Common/Empty'
+import './DelegatedFromUserCard.css'
 
 const SNAPSHOT_SPACE = process.env.GATSBY_SNAPSHOT_SPACE || '' // TODO: Move to snapshot utils file
 const EDIT_DELEGATION_URL = snapshotUrl(`#/delegate/${SNAPSHOT_SPACE}`) // TODO: Move to snapshot utils file
 
 interface Props {
```
Comment (ncomerci) [inline src/components/Delegation/DelegatedFromUserCard.tsx:19]: I think the naming convention we are using for props in the repo is 'ComponentName'+'Props'.

In this case would be `DelegatedFromUserCardProps`

File: src/components/Delegation/DelegatedToUserCard.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+import ActionableLayout from '../Layout/ActionableLayout'
+import VotingPower from '../Token/VotingPower'
+import { VotingPowerListModal } from './VotingPowerListModal'
+import VotingPowerListItem from './VotingPowerListItem'
+import './DelegatedToUserCard.css'
+
+const DISPLAYED_DELEGATIONS = 5
+const OPEN_CALL_FOR_DELEGATES_LINK = 'https://forum.decentraland.org/t/open-call-for-delegates-apply-now/5840/5'
+
+interface Props {
```
Comment (ncomerci) [inline src/components/Delegation/DelegatedToUserCard.tsx:20]: Same here

File: src/pages/balance.css [hunk 5]
```diff
@@ -123,6 +113,10 @@
...
   .ui.container.VotingPowerDetail .ActionableLayout.EstateBalance {
-    margin-top: 0px;
+    margin-top: 0;
   }
-}
\ No newline at end of file
+}
+
+.BalancePage .ActionableLayout__Action {
+  min-height: 35.99px !important;
```
Comment (ncomerci) [inline src/pages/balance.css:121]: This `min-height` caught my attention, is it okay?

**Review by ncomerci — COMMENTED** [review]
I leave some minor comments!

**Review by andyesp — APPROVED** [review]

**Pull request merged**